.github/workflows/docs.yml

```diff
@@ -1,5 +1,8 @@
 name: docs
 on:
+  push:
+    branches:
+      - 'vim-9.1.0147-002'
```

Comment:
👀 

(Convention is that such changes go in a separate `TEST COMMIT DELETE LATER` commit, not squashed in with the actual changes.)

Reply:
Crap I forgot to remove that, sorry. I'll try to remember in the future or at least keep that commit separate, as you suggested

Reply:
No worries; that _is_ something that people do if they need to verify something specific on CI.